### ISSUE TITLE
more than one occlusion image in a single item

### ISSUE THREAD

**Issue by luislopes7** [supermemo/ImageOcclusionEditor#1]
hello sorry for my english, I'm new to github therefore I do not know if asking questions in this place is correct.
I managed to make the imageOcclusion work perfect, what I need is to know if it is possible in a single item that has several images to make the occlusions separately, try to select one by one but that does not make sense since pressing ctlr + win + or create a new item

**Comment by alexis-** [comment]
Hi ! Glad to read that you fixed your problem with running Image Occlusion Editor.

About your question, let me rephrase to make sure that I understand you correctly:
> After opening the occlusion editor (ctrl+win+o), you would like to create one item in SuperMemo for each shape that you drew on the image, as opposed to one item in SuperMemo with all the shapes hiding the information underneath.
>
> That is, each shape would create an item as such:
![](https://raw.githubusercontent.com/glutanimate/image-occlusion-enhanced/master/screenshots/screenshot-io-reviewer.png)
(credit: https://github.com/glutanimate/image-occlusion-enhanced)
>
> This feature is already available in Anki, thanks to Glutaminate's Image Occlusion Editor Enhanced.

Unfortuantly, this isn't possible as of now in Image Occlusion Editor.

It is, however, a feature that I would like to implement in the future. If someone would like to contribute a Pull Request for this, I will gladly accept it.

**Comment by luislopes7** [comment]
hi, not exaclty.
What I want is basically what I describe in the images, to be able to make the extracts of the page of a book, each extract with its corresponding occlusion and to be able to put those two in a single item being able to edit the occlusions independently (this is to select the image I want to edit press ctrl + win + e).
The problem is that when I try to do that, it only recognizes me to edit only one of the masks.


sorry for the tricky question and thanks for u good presiposicion[cid:5bc56056-a8c0-4818-af89-66b9abefd4b6]
[cid:3e124aed-2bce-4675-89f0-f2f7f3a77360]
[cid:f89c0a90-64c4-4717-9fb1-16262f9f2d53]
________________________________
De: Alex Incogito <notifications@github.com>
Enviado: lunes, 22 de abril de 2019 11:02
Para: supermemo/ImageOcclusionEditor
Cc: luislopes7; Author
Asunto: Re: [supermemo/ImageOcclusionEditor] more than one occlusion image in a single item (#1)


Hi ! Glad to read that you fixed your problem with running Image Occlusion Editor.

About your question, let me rephrase to make sure that I understand you correctly:

After opening the occlusion editor (ctrl+win+o), you would like to create one item in SuperMemo for each shape that you drew on the image, as opposed to one item in SuperMemo with all the shapes hiding the information underneath.

That is, each shape would create an item as such:
[https://raw.githubusercontent.com/glutanimate/image-occlusion-enhanced/master/screenshots/screenshot-io-reviewer.png]<https://raw.githubusercontent.com/glutanimate/image-occlusion-enhanced/master/screenshots/screenshot-io-reviewer.png>
(credit: https://github.com/glutanimate/image-occlusion-enhanced)

This feature is already available in Anki, thanks to Glutaminate's Image Occlusion Editor Enhanced.

Unfortuantly, this isn't possible as of now in Image Occlusion Editor.

It is, however, a feature that I would like to implement in the future. If someone would like to contribute a Pull Request for this, I will gladly accept it.

**Comment by alexis-** [comment]
Hi luislope,

I cannot see the pictures that you mentioned.
However I think that I understand what you meant.
You would like to have multiple images, with their occlusions, in a single SuperMemo item. Is that correct ?

At the moment, I do not think that such a feature will be implemented.

SuperMemo advocates one item equals one atomic memory. If you can give describe a situation where this principle is upheld, but multiple occlusions are required in a single item, the feature might be reconsidered.

Sent from ProtonMail mobile

-------- Original Message --------
